common/djangoapps/util/storage.py

```diff
@@ -70,5 +68,8 @@ def resolve_storage_backend(
                 break
             storage_path = storage_path.get(deep_setting_key)
 
-    StorageClass = import_string(storage_path or settings.DEFAULT_FILE_STORAGE)
+    fallback_backend = storages_config.get("default", {}).get(
+        "BACKEND", "django.core.files.storage.FileSystemStorage"
+    )
+    StorageClass = import_string(storage_path or fallback_backend)
```

Comment:
```suggestion
    StorageClass = import_string(storage_path or settings.STORAGES['default']['BACKEND'])
```
STORAGES needs to have a default backend, so we can count on STORAGES having these keys, thus I believe the fallback behavior would be dead code. Unless you know of a case when it's necessary, let's remove the fallback.

Reply:
You're correct! If `STORAGES` is guaranteed to have a default BACKEND, then specifying a fallback default storage becomes redundant. Removed it!